jmh-core/src/main/java/org/openjdk/jmh/util/Optional.java

```diff
@@ -51,6 +52,10 @@ public T orElse(T elseVal) {
         return (val == null) ? elseVal : val;
     }
 
+    public T orElseGet(Supplier<T> alternativeSupplier) {
```

Comment:
I used the [name of the same method from `j.u.Optional`]( https://docs.oracle.com/javase/8/docs/api/java/util/Optional.html#orElseGet-java.util.function.Supplier-), because switching to `j.u.Optional` was the whole idea behind the original state of the PR.